### ISSUE TITLE
Youtube Player API deprecation

### ISSUE THREAD

**Issue by zabojad** [davidohayon669/react-native-youtube#554]
[The Youtube Player API is obsolete and Youtube now requests developpers to use the Youtube IFrame player instead :](https://developers.google.com/youtube/android)

> Note: The [YouTube Android Player API](https://developers.google.com/youtube/android/player) is no longer supported or maintained by YouTube. This SDK should not be used, due to [security vulnerabilities](https://www.cve.org/cverecord?id=CVE-2023-0460). It will be fully deprecated on or after May 1, 2023.

I think we should also make this lib obsolete and invite rect native users to use this lib instead: https://github.com/LonelyCpp/react-native-youtube-iframe

We could add a big note about that in the README and make this repo an archive...

@davidohayon669 What do you think about that?

**Comment by JHM69** [comment]
I have wasted 3 hours because of this. Sad

**Comment by davidohayon669** [comment]
@zabojad should probably do it

**Comment by rklf** [comment]
I agree. Thanks for your hard work @davidohayon669 on this, but it might be misleading from now on. Also it could help to make @LonelyCpp package better by moving contributors there (:
